### PR TITLE
[kubevirt][template-validator]Add test lane for older version of template-validator

### PR DESCRIPTION
**Pull request by vatsalparekh**
Add test lane for older repository of template-validator

This depends on https://github.com/kubevirt/kubevirt-template-validator/pull/65

### CODE REVIEW

File: ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- as: e2e-functests
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: e2e-functests
+      cli: latest
+      commands: |
+        ./hack/env/install-validator.sh
+        export component=template-validator-container
+        export VALIDATOR_IMAGE=$(echo $IMAGE_FORMAT | envsubst)
```
Comment (akrejcir) [inline ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml:39]: The script `./hack/env/install-validator.sh` will use `$VALIDATOR_IMAGE`, so it needs to be called after the export.

Comment (vatsalparekh) [reply ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml:38]: fixed

File: ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
```
Comment (petr-muller) [inline ci-operator/config/kubevirt/kubevirt-template-validator/kubevirt-kubevirt-template-validator-master.yaml:18]: `tag_specification` is legacy and should not be used except for configs with legacy tests. 

Please use `releases` instead: https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release

**Pull request merged**